services/web/server/src/simcore_service_webserver/director_v2.py

```diff
@@ -73,8 +80,9 @@ async def create_or_update_pipeline(
         )
         return computation_task_out
 
-    except _DirectorServiceError:
+    except _DirectorServiceError as e:
         log.error("could not create pipeline from project %s", project_id)
+        raise e
```

Comment:
It's odd that the tests seem to pass with this here in place. Please do let me know on how to handle this.

Reply:
I remember talking extensively with @pcrespov about it.. will try to find it again

Reply:
can you revert back this change please? I see while testing your PR some errors in the frontend that were not happening before. 
to reproduce, just create a pipeline, start it, and move one of the node. then you will get errors all the time during the time the pipeline is run cause the director2 will return a 403-forbidden since you cannot change a running pipeline.